erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -199,8 +199,8 @@ def validate_bank_accounts(self):
 
 	def validate_account_type(self, account, account_types):
 		account_type = frappe.db.get_value("Account", account, "account_type")
-		if account_type not in account_types:
-			frappe.throw(_("Account Type for {0} must be {1}").format(account, comma_or(account_types)))
+		# if account_type not in account_types:
+		# 	frappe.throw(_("Account Type for {0} must be {1}").format(account, comma_or(account_types)))
```

Comment:
Can we get rid of this method altogether given it just adds a DB call and does no validation anymore? [it's the same in v13, v14, develop]